packages/frontend-dynamic-feature-loader/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@backstage/frontend-dynamic-feature-loader",
+  "version": "0.0.1-next.0",
```

Comment:
```suggestion
  "version": "0.0.0",
```

and combine it with a `minor` level changeset

Reply:
Done in commit 7452c175b70c1df851251dfa7adc582d529bb5de